### ISSUE TITLE
Why Twig is required?

### ISSUE THREAD

**Issue by TrogWarZ** [intaro/pinba-bundle#9]
Hi!

I'm using Symfony 3.x as json-api without need for twig at all.

After installing this bundle i see error `Fatal error: Uncaught Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: The service "fos_rest.exception.twig_controller" has a dependency on a non-existent service "twig".`

So, why? And how to disable this requirement? I don't need any rendering stuff from Pinba – only "smart timers" to see queries, correct routes etc.

**Comment by muxx** [comment]
You are right. I think it is good idea to make all types of timers in bundle optional.

**Comment by TrogWarZ** [comment]
@muxx, yeah, it will be good. I don't have any experience in writing bundles, but i can try to make a PR if you don't have enough time.

**Comment by muxx** [comment]
It will be great if you prepare PR! 👍
